### DISCUSSION TITLE
How to configure order of directories

### DISCUSSION THREAD

**Discussion by embostan** [react-cosmos/react-cosmos#1650] [Q&A]
Hello, inside of the directory containing fixtures, I havre several subdirectories which allow to group fixtures and MDX docu (e.g. "Getting started", "Components"...).
However by default the ordering is alphabetical, which isn't ideal (Getting started should be first).

How do I configure a custom ordering? Thanks

**Comment by ovidiuch** [comment]
Hi @StanLaurent, this isn't supported at the moment.

It's not pretty and kinda obvious but you could prefix the important files with numbers or an underscore :).

Cosmos also has a Bookmark feature, which pins some fixtures above the full fixture tree (which is similar to a file explorer).

Hope that helps. Let me know what you envision. Not sure if we'll be able to accommodation but I'm interested what you have in mind.
